ironfish-cli/src/commands/accounts/pay.ts

```diff
@@ -150,6 +150,11 @@ export class Pay extends IronfishCommand {
       this.exit(0)
     }
 
+    if (!isValidPublicAddress(to)) {
+      this.log(`A valid public address is required`)
+      this.exit(0)
```

Comment:
This is an error so it should exit 1

```suggestion
      this.exit(1)
```